### ISSUE TITLE
[CRASH] When system object is assigned to any facet of a face

### ISSUE THREAD

**Issue by hiiamboris** [red/red#4461]
Keyword: GUI, View, VID, layout

**Describe the bug**

Try: `layout [base data system]`
Or: `layout [base data (system/words)]`

Both result in:
```
*** Runtime Error 19: stack error or overflow                      
*** in file: /D/devel/red/red-src/red-master/runtime/hashtable.reds
*** at line: 700                                                   
***                                                                
***   stack: red/_hashtable/get-value 00A93138h 11061468           
***   stack: red/ownership/unbind 02E01C28h                        
***   stack: red/ownership/unbind 02E01C28h                        
***   stack: red/ownership/unbind 02CDB7D0h                        
***   stack: red/ownership/unbind 02CDB5E0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
***   stack: red/ownership/unbind 02CDB5F0h                        
***   stack: red/ownership/unbind 02C41160h                        
```

**Expected behavior**

`data` facet assigned to the given object

**Platform version (please complete the following information)**
```
Red 0.6.4 for Windows built 20-May-2020/11:46:36+03:00
```

**Comment by hiiamboris** [comment]
Not layout is the culprit, but ownership. Assign **almost any facet** so `system` and it crashes when tries to make this face the owner of `system`:
```
f: make-face 'text
f/pane: system
```
(same crash log)
`extra` is unaffected as it gets special treatment - totally ignored by on-change
Some facets crash when you *unassign* them (`actors`, `edge`):
```
f/actors: system
f/actors: none
```
(same crash log)

**Comment by 9214** [comment]
> extra is unaffected as it gets special treatment - totally ignored by on-change

Worth documenting, I puzzled over `extra`'s inertness not so recently too.

**Comment by hiiamboris** [comment]
Maybe in official docs on View?

**Comment by dockimbel** [comment]
FIxed by the fix for #4546.
